.github/workflows/action-test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test github-action-copy-files
+
+on:
+  ['push']
```

Comment:
we probably don't want to run this on each push, do we? this means any push to any branch